src/cmd-kola

```diff
@@ -46,14 +46,22 @@ if os.path.isfile(old_denylist_path):
     print(f"⛔ Detected an old filename for denylist; please update your filename to `kola-denylist.yaml`")
     denylist_path = old_denylist_path
 
+current_branch = subprocess.check_output(["git", "-C", "src/config", "branch",
+                                          "--show-current"],
+                                         encoding='utf-8').strip()
```

Comment:
We often run tests on our own branches in our own forks.. would it be better to pick up and switch on `fedora-coreos.stream` from manifest.yaml instead? I'm currently working in the `dusty-next-f33` off my fork so I don't know if this will be what we want.

Reply:
suggestion: use the keyword `stream` instead of `branch`.